style.css

```diff
@@ -0,0 +1,598 @@
+  margin-left: 55%;
+  padding-top: 10px;
+}
+
+.desc-left {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-right: 55%;
+  padding-top: 10px;
+}
+
+.desc-left-1 {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-right: 5%;
+  padding-top: 10px;
+}
+
+.skills {
+  display: flex;
+  gap: 30px;
+  width: 29px;
+  height: 16px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  margin-left: 55%;
+  padding-top: 15px;
+}
+
+.skills-left {
+  display: flex;
+  gap: 20px;
+  width: 29px;
+  height: 16px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  margin-right: 55%;
+  padding-top: 15px;
+}
+
+.text {
+  color: #6070ff;
+  text-align: center;
+  border-radius: 10px;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.02em;
+  background: #ebebff;
+  height: 24px;
+  border: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 20px;
+}
+
+.text-ruby {
+  color: #6070ff;
+  text-align: center;
+  border-radius: 80px;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 19px;
+  letter-spacing: 0.04em;
+  background: #ebebff;
+  height: 24px;
+  border: none;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  padding: 10px 20px;
+  width: fit-content;
+}
+
+.btn {
+  padding-top: 80px;
+  margin-right: 312px;
+  background-color: white;
+  color: #396df2;
+  font-size: 1px;
+  cursor: pointer;
+}
+
+.btn2 {
+  padding-top: 80px;
+  margin-right: 948px;
+  background-color: white;
+  color: #396df2;
+  font-size: 1px;
+  cursor: pointer;
+}
+
+.see-more {
+  font-weight: 600;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  cursor: pointer;
+}
+
+.text-1 {
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  border-radius: 10px;
+  border: 1px solid #396df2;
+  align-items: center;
+  letter-spacing: 0.01em;
+  width: 120px;
+  height: 44px;
+  background-color: white;
+  color: #396df2;
+}
+
+.text-2 {
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  border-radius: 10px;
+  background: #fff;
+  border: 1px solid #396df2;
+  align-items: center;
+  letter-spacing: 0.01em;
+  width: 120px;
+  height: 44px;
+  background-color: #4053fc;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  color: #fff;
+}
+
+.text-3 {
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  border-radius: 10px;
+  border: 1px solid #396df2;
+  letter-spacing: 0.01em;
+  width: 140px;
+  height: 44px;
+  background-color: white;
+  color: #396df2;
+}
+
+.card-info-2 {
+  width: 100%;
+  height: 500px;
+  background-color: var(--main-bg-color);
+  border: 1px solid #dfe1e6;
+  align-items: center;
+  justify-content: center;
+  padding-bottom: 2px;
+  border-top-right-radius: 130px;
+}
```

Comment:
Nice job adding the About section to your project 😄! Please consider the following observations to improve your design 😉:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/207970308-32c9db23-f315-4f19-8bb0-dd00d894c32b.png)  | ![image](https://user-images.githubusercontent.com/42799579/207970269-85be2b23-42ab-44e7-a2c0-2719a44953ef.png)  |

- [ ] Please update the color of the 'About Myself' header.

- [ ] Kindly notice that your project is missing the languages icons 😉.

- [ ] Please remove the `bold` weight from your 'Languages', 'Frameworks', and 'Skills' headers.

- [ ] Make sure to update the gap between the 'Languages', 'Frameworks', and 'Skills' sections.

- [ ] Make sure to update the gap between your 'Get My Resume' button & your social links 😄.

- [ ] Please increase your 'Get My Resume' button padding.

By updating this, your project will look closer to the original design! Remember: this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.